insights/Extract_MC_Insights_DBX.py

```diff
@@ -9,19 +9,55 @@
 
 # COMMAND ----------
 
-# DBTITLE 1,Runtime Variables
```

Comment:
Can we add usage instructions to this notebook?

This should also include a callout that script creates (and replaces) tables.

Reply:
Nit - Can we lowercase the filename for consistency? Don't forget we have to update the link [here](https://github.com/monte-carlo-data/monte-carlo-python-sdk-examples/blob/main/README.md)

Reply:
done and done